Gemfile

```diff
@@ -42,6 +42,8 @@ gem 'paranoia'
 gem 'sentry-raven'
 gem 'inline_svg'
 gem 'paper_trail'
+gem 'telegram-bot'
```

Comment:
is this working correctly? the doc says `require 'telegram/bot'`

Reply:
Yes.
`require 'telegram/bot'` it for non rails application

Reply:
I think works ;) based on the screenshot above